### PR TITLE
Bratseth/remove dependencies

### PR DESCRIPTION
**Pull request by bratseth**
@vekterli please review

This removes all Vespa deps. This was supposed to be completely standalone, but dependencies have leaked in. I've now removed them because that will make it simpler to provide a java8 build of this.

### CODE REVIEW

File: vespajlib/src/main/java/com/yahoo/collections/Tuple2.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * new class and do the work inside that class instead? Etc.)
  * </p>
  *
- * @author <a href="mailto:steinar@yahoo-inc.com">Steinar Knutsen</a>
+ * @authorSteinar Knutsen
```
Comment (vekterli) [inline vespajlib/src/main/java/com/yahoo/collections/Tuple2.java]: Missing a space, which has also angered the javadoc CI build gods

**Review by vekterli — APPROVED** [review]
:+1: see comment re: Travis failure

**Comment by hmusum** [comment]
Breaks vespa-http-client-extensions:
`[ERROR] /var/builds/workspace/115429-v3-component-build/BUILD_CONTAINER/build/label/DOCKER-HIGHCPU/app_root/vespa-http-client-extensions/src/main/java/com/yahoo/vespa/http/client/extensions/package-info.java:[7,33] package com.yahoo.api.annotations does not exist`

**Comment by bratseth** [comment]
Thanks. Fixing ...

**Pull request merged**